pkg/cmd/codespace/select.go

```diff
@@ -0,0 +1,68 @@
+package codespace
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+type selectOptions struct {
+	filePath string
+}
+
+func newSelectCmd(app *App) *cobra.Command {
+	opts := selectOptions{}
+
+	selectCmd := &cobra.Command{
+		Use:   "select",
+		Short: "Select a codespace",
+		Hidden: true,
```

Comment:
The only thing that I'm 🤔 about is if we really do want this hidden. I realize it's an extension utility type of command, but how will extension developers find out about it?

Reply:
Yeah, that sounds good, but I think the docs at https://cli.github.com/manual/index are auto-generated from the code, and not sure if they pick up hidden commands? cc @mislav

Reply:
Just checked locally - looks like it does not pick up the hidden commands:

<img width="1904" alt="image" src="https://user-images.githubusercontent.com/1478800/161606202-b81d62ca-5969-4af1-9658-23fcb3f376f1.png">